src/js/helpers/storage_init.js

```diff
@@ -0,0 +1,34 @@
+var storage_module = null;
+var local_storage = require('./local_storage'),
+	session_storage = require('./session_storage'),
+	single_cookie = require('./single_cookie'),
+	cookies       = require('./cookies');
+
+module.exports = {
+	validateType: function( storage_type ) {
+		// Default to valid_values[0] if storage_type is not in valid_values
+		var valid_values = ['cookies', 'singleCookie', 'localStorage', 'sessionStorage'];
+		return valid_values.indexOf( storage_type ) > -1 ? storage_type : valid_values[0];
+	},
+	set: function( storage_type, session_length ) {
+		storage_type = this.validateType( storage_type );
+		switch ( storage_type ) {
+			case 'singleCookie':
+				storage_module = single_cookie;
+				break;
+			case 'localStorage':
+				storage_module = local_storage;
+				storage_module.setSessionLength( session_length );
+				break;
+			case 'sessionStorage':
+				storage_module = session_storage;
+				break;
+			case 'cookies':
+			default:
+				storage_module = cookies;
```

Comment:
Specifying `break` for default statement. Some mention it is a good practice. See [Break on default case in switch](https://softwareengineering.stackexchange.com/questions/201777/break-on-default-case-in-switch/6969).

```suggestion
			case 'cookies':
			default:
				storage_module = cookies;
				break;
```